src/ia-menu-slider.js

```diff
@@ -18,6 +18,7 @@ export class IAMenuSlider extends LitElement {
       open: { type: Boolean },
       selectedMenu: { type: String },
       animateMenuOpen: { type: Boolean },
+      manuallyHandleClose: { type: Boolean },
```

Comment:
Do we have multiple consumers of this or is the only one `ItemNavigator.js` in petabox? If it's the only consumer, I think you can just make this behavior the default and always let the consumer handle the close event.

Reply:
only the item nav is consuming this as of right now.  I have a fast follow after this and will address then. thanks for raising, too.  the grey area between to use a toggle and not to toggle can get vast, and I appreciate the your beacon/call to keep it simple 😃